apps/dashboard/src/components/primitives/form/form.tsx

```diff
@@ -101,28 +101,35 @@ const formMessageVariants = cva('flex items-center gap-1', {
   },
 });
 
-const FormMessage = React.forwardRef<HTMLParagraphElement, React.HTMLAttributes<HTMLParagraphElement>>(
-  ({ className, children, ...props }, ref) => {
-    const { error, formMessageId } = useFormField();
-    const body = error ? String(error?.message) : children;
+const FormMessagePure = React.forwardRef<
```

Comment:
The `FormMessagePure` is a label + icon that is not dependent on the `FormField`, it's used in a few cases.